src/main/java/br/com/metricminer2/domain/Commit.java

```diff
@@ -100,12 +101,17 @@ public Calendar getDate() {
 		return date;
 	}
 
-	public void addBranch(String branch) {
+	public void addBranch(String branch, String masterBranch) {
```

Comment:
On one hand, I see the value of having this `isInMasterBranch()` method here. On the other hand, I am not sure if that's the most elegant way to do it. 

Seems that this logic (of comparing the branch + setting the boolean) should exist in the factory of this class, thus, in GitRepository. So, what about receiving this `masterBranch` as a boolean in the constructor (all `Commit`s now will require it) and `GitRepository` sets it?